gprofiler/utils/__init__.py

```diff
@@ -148,7 +148,9 @@ def start_process(
         else:
             # explicitly remove our directory from LD_LIBRARY_PATH
             env = env if env is not None else os.environ.copy()
-            env.update({"LD_LIBRARY_PATH": ""})
+            if "TMPDIR" not in env and "TMPDIR" in os.environ:
+                env["TMPDIR"] = os.environ.get("TMPDIR")
```

Comment:
```suggestion
                env["TMPDIR"] = os.environ["TMPDIR"]`
```
makes it clear that the value can't be `None`.